internal/transport/handler_server.go

```diff
@@ -236,7 +239,7 @@ func (ht *serverHandlerTransport) WriteStatus(s *Stream, st *status.Status) erro
 			})
 		}
 	}
-	ht.Close()
+	ht.Close(fmt.Errorf("finished writing status"))
```

Comment:
`finished writing status`. 

Why is this an error?

Is this error propagated to the user? Or is this used by another component, and if so how?